src/integrationTest/java/uk/gov/hmcts/reform/sscs/controllers/SscsBulkScanValidateRecordCallback.java

```diff
@@ -162,8 +186,7 @@ public void should_return_error_when_appellant_and_appointee_details_are_only_pa
                 "Appointee last name is empty",
                 "Appellant title is empty",
                 "Appellant first name is empty",
-                "Appellant last name is empty",
-                "Hearing option telephone, video and face to face are empty. At least one must be populated");
```

Comment:
Isn't there a valid scenario that can be tested here for the line you have removed? Maybe a new test?